src/js/components/Header.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+class Header extends React.Component {
```

Comment:
wait, i've got one comment. let's rename this component into something else other than `Header` because it's really undescriptive